fetch/api/basic/scheme-blob.sub.any.js

```diff
@@ -45,4 +45,44 @@ invalidRequestMethods.forEach(function(method) {
 checkKoUrl("blob:not-backed-by-a-blob/", "GET",
            "Fetching [GET] blob:not-backed-by-a-blob/ is KO");
 
+var empty_blob = new Blob([]);
+checkFetchResponse(URL.createObjectURL(empty_blob), "", "", 0,
+                  "Fetching URL.createObjectURL(empty_blob) is OK");
+
+var empty_type_blob = new Blob([], {type: ""});
```

Comment:
Note that these tests do not include any content, so a engine with an algorithm that tries to sniff the content type will pass this, but in some cases I _think_ would provide a potentially non-standard content type.

Reply:
Blob content should never be sniffed to determine the response `Content-Type` header though. So maybe we should include a test where the blob starts with a file signature of a JPEG or some such.